app/assets/javascripts/edit.js

```diff
@@ -1,6 +1,6 @@
 function editText(){
     thisElement = $(this)
-    $(this).keypress(function(event) {
+    thisElement.keypress(function(event) {
```

Comment:
This name should probably match the `$` style - the name isn't much clearer than just using `$(this)` and since you don't use it more than once, you might want to just leave this as it is.